meetings/2024-10-09.md

```diff
@@ -0,0 +1,42 @@
+# Node.js  TypeScript team Meeting 2024-10-09
+
+## Links
+
+* **Recording**: <https://www.youtube.com/watch?v=ywOlXuB6BNA>
+* **GitHub Issue**: <https://github.com/nodejs/typescript/issues/12>
+
+## Present
+
+* Typescript team: @nodejs/typescript
+* Marco Ippolito: @marco-ippolito
+* Jake Bailey: @jakebailey
+* Carlos Espa: @Ceres6
+
+## Agenda
+
+Jake: Extension rewriting merged, maybe will be released in TypeScript 5.7-
+ Should work fine with –experimental-strip-types.
+Jakob: Codemod for transforming extensions from ‘.js’ to ‘.ts’ is almost ready.
+
+## Announcements
+
+* Extracted from **typescript-agenda** labelled issues and pull requests from the **nodejs org** prior to the meeting.
+
+### nodejs/loaders
+
+* Roadmap for experimental TypeScript support [#217](https://github.com/nodejs/loaders/issues/217)
+
+Marco: new API to perform transpilation the same way Node does in module <https://github.com/nodejs/node/pull/55282>
+Marco: For the future having a flag TS flag compatible <https://github.com/microsoft/TypeScript/issues/59601> with experimental-strip-types would make it easier to unflag
+Jake: Not possible for 5.7 beta is already cut, discussion ongoing. Maybe some details will be different from the current strip-types implementation <https://github.com/microsoft/TypeScript/issues/59601>
+Marco: As soon as 5.7 is out will be adding documentation to the website.
+Jake: 5.7 documentation will contain details on what the new flag looks like and caveats.
+Marco: Moving from early development to experimental.
```

Comment:
Can these be bulleted? This all gets rendered as a single paragraph in markdown.